Source/GSTitleView.m

```diff
@@ -238,9 +239,24 @@ - (void) mouseDown: (NSEvent*)theEvent
   NSDate   *theDistantFuture = [NSDate distantFuture];
   NSPoint  startWindowOrigin;
   NSPoint  endWindowOrigin;
+  CGFloat  leftLimit;
+  CGFloat  topLimit;
+  CGFloat  rightLimit;
+  CGFloat  bottomLimit;
 
   NSDebugLLog (@"NSMenu", @"Mouse down in title!");
 
+  // Define move constrains for menu
+  if (_ownedByMenu)
+    {
+      NSRect screenFrame = [[NSScreen mainScreen] frame];
```

Comment:
As always we should protect the code against _window and [_window screen] being nil. That would result in a crash on some machines.
And maybe it would be worthwhile to save [_window frame] in a local variable.

Reply:
Thanks, done.